final-project/core/backend/app/routers/kanji.py

```
@@ -0,0 +1,123 @@
+    query = select(Kanji).filter(Kanji.id == kanji_id)
+    result = await db.execute(query)
+    kanji = result.scalar()
+    
+    if not kanji:
+        raise HTTPException(status_code=404, detail="Kanji not found")
+    
+    # Get review stats
+    stats_query = (
+        select(
+            func.count(ReviewItem.id).label("total_reviews"),
+            func.sum(case((ReviewItem.correct == True, 1), else_=0)).label("correct_reviews")
```

Ruff (0.8.2)

Avoid equality comparisons to `True`; use `if ReviewItem.correct:` for truth checks

Replace with `ReviewItem.correct`

(E712)